### PR TITLE
WIP: kustomize: adding CRDs support

### PR DESCRIPTION
**Pull request by danmx**
Signed-off-by: danmx <daniel@iziourov.info>

## Description

Support Kustomize's CRDs [configuration](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/crds/).

## How Has This Been Tested?

Tested in a private repository.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ When you run `bazel run ///helloworld:mynamespace.apply`, it applies this file i
 | ***name_prefix***         | `None`         | Adds prefix to the names of all resources defined in manifests.
 | ***name_suffix***         | `None`         | Adds suffix to the names of all resources defined in manifests.
 | ***patches***             | `None`         | A list of patch files to overlay the base manifests. See [Base Manifests and Overlays](#base-manifests-and-overlays).
+| ***crds***                | `None`         | A list of custom resource definitions (CRDs) files. The presence of this field is to allow kustomize be aware of CRDs and apply proper transformation for any objects in those types.
```
Comment (kzadorozhny) [inline README.md:92]: Please link https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/crds/ page in the parameter description.

**Comment by michaelschiff** [comment]
@danmx Thanks for the contribution! Please add a test for this new functionality in `skylib/kustomize/tests`

**Comment by danmx** [comment]
it seems in the end it doesn't work the way I was initially thinking. I'll have to work a little bit more on it.

**Comment by danmx** [comment]
recommended way is to use configs: https://github.com/kubernetes-sigs/kustomize/issues/480#issuecomment-431083636
Custom schemas are still in KEP: https://github.com/kubernetes/enhancements/issues/2206

**Pull request closed without merging**